util/ssh_client.go

```diff
@@ -2,14 +2,36 @@ package util
 
 import (
 	"code.google.com/p/go.crypto/ssh"
+	"errors"
+	"github.com/pkg/sftp"
```

Comment:
Next time, please organize these imports. Put Go imports together, github, and ours together. See other examples from my code. Thanks.